### PR TITLE
RangeSlider: add Angular, Vue and React code snippets

### PR DESCRIPTION
**Pull request by ksercs**

### CODE REVIEW

File: concepts/05 Widgets/RangeSlider/00 Overview.md [hunk 1]
```diff
@@ -17,8 +20,114 @@ The following code adds a simple **RangeSlider** to your page. The **start** and
         });
     });
 
+##### Angular
+
+    <!--HTML-->
+    <dx-range-slider
+        [min]="0"
+        [max]="100"
+        [(start)]="startValue"
```
Comment (AlexJameson) [inline concepts/05 Widgets/RangeSlider/00 Overview.md:29]: 1. Do we really need two-way binding here?
2. Why did not you specify the start and end values right in markup?

File: concepts/05 Widgets/RangeSlider/05 Customize Widget Appearance.md [hunk 2]
```diff
@@ -14,8 +17,97 @@ The **RangeSlider** can display labels for the [min](/api-reference/10%20UI%20Wi
         });
     });
 
+##### Angular
+
+    <!--HTML-->
+    <dx-range-slider
```
Comment (AlexJameson) [inline concepts/05 Widgets/RangeSlider/05 Customize Widget Appearance.md]:
```suggestion
    <dx-range-slider ...
```

Comment (AlexJameson) [reply concepts/05 Widgets/RangeSlider/05 Customize Widget Appearance.md]: Here and below

File: concepts/05 Widgets/RangeSlider/05 Customize Widget Appearance.md [hunk 3]
```diff
@@ -41,6 +225,66 @@ To specify whether or not the selected range should be highlighted, use the **sh
...
+        [showRange]="false"
```
Comment (AlexJameson) [inline concepts/05 Widgets/RangeSlider/05 Customize Widget Appearance.md]:
```suggestion
        [showRange]="false">
```

**Review by AlexJameson — CHANGES_REQUESTED** [review]

File: concepts/05 Widgets/RangeSlider/05 Customize Widget Appearance.md (hunk 2 above)
Comment (RomanTsukanov) [inline concepts/05 Widgets/RangeSlider/05 Customize Widget Appearance.md]:
```suggestion
    <dx-range-slider ... >
```

**Review by RomanTsukanov — CHANGES_REQUESTED** [review]

File: concepts/05 Widgets/RangeSlider/00 Overview.md (hunk 1 above)
Comment (ksercs) [reply concepts/05 Widgets/RangeSlider/00 Overview.md:29]: 1. Yes, `start` and `end` values indicate the position of the left and right sliders of the changing range
2. I can do it, but these values can be changed both from the code and by user in UI. Besides, it has already been written in Angular approach before my changes

File: concepts/05 Widgets/RangeSlider/05 Customize Widget Appearance.md (hunk 2 above)
Comment (ksercs) [reply concepts/05 Widgets/RangeSlider/05 Customize Widget Appearance.md]: fixed

File: concepts/05 Widgets/RangeSlider/05 Customize Widget Appearance.md (hunk 3 above)
Comment (ksercs) [reply concepts/05 Widgets/RangeSlider/05 Customize Widget Appearance.md]: fixed

File: concepts/05 Widgets/RangeSlider/05 Customize Widget Appearance.md (hunk 2 above)
Comment (ksercs) [reply concepts/05 Widgets/RangeSlider/05 Customize Widget Appearance.md]: fixed

**Review by RomanTsukanov — APPROVED** [review]

**Review by AlexJameson — APPROVED** [review]

**Pull request merged**